modules/cnf-topology-aware-lifecycle-manager-operator-and-platform-update.adoc

```diff
@@ -0,0 +1,136 @@
+
+. Apply the prep work for the platform and the Operator update.
+
+.. Save the content of the `ClusterGroupUpgrade` CR with the policies for platform update preparation work, catalog source updates, and target clusters to the `cgu-platform-operator-upgrade-prep.yml` file, for example:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu-platform-operator-upgrade-prep
+  namespace: default
+spec:
+  managedPolicies:
+  - du-upgrade-platform-upgrade-prep
+  - du-upgrade-operator-catsrc-policy
+  clusterSelector:
+  - group-du-sno
+  remediationStrategy:
+    maxConcurrency: 10
+  enable: true
+----
+
+.. Apply the `cgu-platform-operator-upgrade-prep.yml` file to the hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f cgu-platform-operator-upgrade-prep.yml
+----
+
+.. Monitor the process. Upon completion, ensure that the policy is compliant by running the following command:
++
+[source,terminal]
+----
+$ oc get policies --all-namespaces
+----
+
+. Create the `ClusterGroupUpdate` CR for the platform and the Operator update with the `spec.enable` field set to `false`.
+.. Save the contents of the platform and Operator update `ClusterGroupUpdate` CR with the policies and the target clusters to the `cgu-platform-operator-upgrade.yml` file, as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu-du-upgrade
+  namespace: default
+spec:
+  managedPolicies:
+  - du-upgrade-platform-upgrade <1>
+  - du-upgrade-operator-catsrc-policy <2>
+  - common-subscriptions-policy <3>
+  preCaching: true
+  clusterSelector:
+  - group-du-sno
+  remediationStrategy:
+    maxConcurrency: 1
+  enable: false
+----
+<1> This is the platform update policy.
+<2> This is the policy containing the catalog source information for the Operators to be updated. It is needed for the pre-caching feature to determine which Operator images to download to the spoke cluster.
+<3> This is the policy to update the Operators.
+
+.. Apply the `cgu-platform-operator-upgrade.yml` file to the hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f cgu-platform-operator-upgrade.yml
+----
+
+. Optional: Pre-cache the images for the platform and the Operator update.
+.. Enable pre-caching in the `ClusterGroupUpgrade` CR by running the following command:
++
+[source,terminal]
+----
+$ oc --namespace=default patch clustergroupupgrade.ran.openshift.io/cgu-du-upgrade \
+--patch '{"spec":{"preCaching": true}}' --type=merge
+----
+
+.. Monitor the update process and wait for the pre-caching to complete. Check the status of pre-caching by running the following command on the spoke cluster:
++
+[source,terminal]
+----
+$ oc get jobs,pods -n openshift-talo-pre-cache
```

Comment:
This should be changed to talm.
@nishant-parekh could you please confirm you will change this in the repo?

Reply:
that is correct this should be changed to openshift-talm-pre-cache